### PR TITLE
Added help resources test case

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary

Ref: https://github.com/mattermost/mattermost-webapp/pull/12026

- [x] Browser
- [x] Mobile App (version, OS)
- [x] Desktop App (version, OS)

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/68"></a>

### CODE REVIEW

File: data/test-cases/suite/system-console/site-configuration/customization/MM-Test-Case-1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+2. Go to System Console > Customization
+3. Observe **Help Link**
+4. Clear **Help Link** field
+5. Go to Channel
+6. Click **Help** icon
+
+**Expected**
+
+2. Customization page should be displayed
+3. ***Help Link** should be empty
```
Comment (jgilliam17) [inline data/test-cases/suite/system-console/site-configuration/customization/MM-Test-Case-1.md:47]:
```suggestion
3. **Help Link** should be empty
```

File: data/test-cases/suite/system-console/site-configuration/customization/MM-Test-Case-1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+4. **Help Resources** option should not be exist
```
Comment (jgilliam17) [inline data/test-cases/suite/system-console/site-configuration/customization/MM-Test-Case-1.md:48]:
```suggestion
4. **Help Resources** option should not exist
```

**Review by jgilliam17 — CHANGES_REQUESTED** [review]
Just two suggestions, otherwise looks good.

**Review by yasserfaraazkhan — APPROVED** [review]

**Comment by saturninoabril** [comment]
Test case saved - MM-T5383

**Pull request merged**